project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -165,6 +165,16 @@ class PreferenceForm extends React.Component {
           restaurants = restaurants.concat(restaurant.results);
         }
 
+        if (restaurants.length === 0) {
+          let message = "We can't find any restaurants near you";
+          if (this.state.open) {
+            message += ' that are currently open';
+          }
+          message += '. Please try changing your preferences.';
```

Comment:
nit: lets make this 
+= ' that match your preferences. Please try changing your preferences' 
(im just being pedantic about adding context to the cant find restaurants near you 🙃  )